xml/System.Security.Cryptography/ECDiffieHellman.xml

```diff
@@ -1080,10 +1090,20 @@ The algorithm-specific key import failed.</exception>
         <Parameter Name="password" Type="System.ReadOnlySpan&lt;System.Char&gt;" Index="1" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="input">To be added.</param>
-        <param name="password">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="input">The PEM text of the encrypted key to import.</param>
+        <param name="password">The password to use for decrypting the key material.</param>
+        <summary>Imports an encrypted RFC 7468 PEM-encoded private key, replacing the keys for this object.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+ When the base-64 decoded contents of `input` indicate an algorithm that uses PBKDF1 (Password-Based Key Derivation Function 1) or PBKDF2 (Password-Based Key Derivation Function 2), the password is converted to bytes via the UTF-8 encoding.
+  Unsupported or malformed PEM-encoded objects will be ignored. If multiple supported PEM labels are found, an exception is thrown to prevent importing a key when the key is ambiguous.
+ This method supports the <c>ENCRYPTED PRIVATE KEY</c> PEM label.
```

Comment:
```suggestion
 When the base-64 decoded contents of `input` indicate an algorithm that uses PBKDF1 (Password-Based Key Derivation Function 1) or PBKDF2 (Password-Based Key Derivation Function 2), the password is converted to bytes via the UTF-8 encoding.
  Unsupported or malformed PEM-encoded objects will be ignored. If multiple supported PEM labels are found, an exception is thrown to prevent importing a key when the key is ambiguous.
 This method supports the `ENCRYPTED PRIVATE KEY` PEM label.
```